volume/mounts/linux_parser_test.go

```diff
@@ -67,12 +67,12 @@ func TestLinuxParseMountRaw(t *testing.T) {
 		"/path:shared":                    "invalid volume specification",
 		"/path:slave":                     "invalid volume specification",
 		"/path:private":                   "invalid volume specification",
-		"name:/absolute-path:shared":      "invalid volume specification",
-		"name:/absolute-path:rshared":     "invalid volume specification",
-		"name:/absolute-path:slave":       "invalid volume specification",
-		"name:/absolute-path:rslave":      "invalid volume specification",
-		"name:/absolute-path:private":     "invalid volume specification",
-		"name:/absolute-path:rprivate":    "invalid volume specification",
+		"name:/absolute-path:shared":      "can't set propagation mode",
+		"name:/absolute-path:rshared":     "can't set propagation mode",
```

Comment:
For others; the full error will be prefixed with `invalid mount config for type <type>:` (I went looking for that, as only "can't set propagation mode" felt to be missing context)